### DISCUSSION TITLE
Logger rest response

### DISCUSSION THREAD

**Discussion by shay-aharonson** [TypedProject/ts-express-decorators#2720] [Q&A]
Hi, 

Apologies in advance for the silly question I'm fairly new to Node.

I've seen the documentation for configuring the logger to display request params, I've tested it out and it works great. I've noticed that it also supports the request "end" state and logs the status code but I did not see any mention of anything in regards to response body, is this supported in the config? Would it require overriding or writing a separate middleware? How would one go about adding that information to the log?

Thanks.

**Comment by Romakita** [comment]
Hello @shay-aharonson 

Here is the documentation: https://tsed.io/docs/logger.html#features

For a performance reason, by default the context log request doesn't display the request & response payload. It's only display when you have an error.

The documentation explain how the log can be customized. 

See you
